CHANGELOG.rst

```diff
@@ -4,10 +4,13 @@ CHANGELOG
 This document describes changes between each past release.
 
 
-0.3.0 (unreleased)
+0.3.0 (2015-09-21)
 ==================
 
-- Nothing changed yet.
+- Separate SyncClient and TokenserverClient code
+- Add a parameter to configure the Tokenserver credentials validity duration
+- Add parameter to be able to create a SyncClient with existing credentials
```

Comment:
Add a parameter to create a SyncClient with already fetched TokenServer credentials?